docs/commands/ts.info.md

```diff
@@ -47,7 +47,9 @@ Each item (per chunk) will contain:
 #### `TS.INFO` Example
 
 ```sql
+
 TS.INFO temperature:2:32
+[b'totalSamples', 3, b'memoryUsage', 4248, b'firstTimestamp', 1, b'lastTimestamp', 5000, b'retentionTime', 0, b'chunkCount', 1, b'chunkSize', 4096, b'chunkType', b'compressed', b'duplicatePolicy', None, b'labels', [], b'sourceKey', None, b'rules', [[b't1_MAX_1000_500', 1000, 500]]]
```

Comment:
@OfirMos should be pretty-printed if possible

Reply:
@filipecosta90 It was already like this, this PR is just adding it to the ts.info command